src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import tasks.*;
```

Comment:
Understand that you need to import multiple classes from Task packages. However, import tasks.* has to be stated explicitly, such as import tasks.Task.